app/assets/javascripts/admin/addon/components/admin-config-areas/about.gjs

```diff
@@ -0,0 +1,15 @@
+import Component from "@glimmer/component";
+import AdminConfigAreasBase from "./base";
+
+export default class AdminConfigAreasAbout extends AdminConfigAreasBase {
```

Comment:
I see what you are aiming for here and I appreciate it, but I think maybe we should hold off from making a base class for now until we've done 2-3 config areas in the UI, since the whole concept is still in draft.

Internally, /t/129128/2 describes a general `AdminConfigAreaCard` reusable component that might be a better building block to start with. We just don't really know what the help inset will be like yet.

Reply:
Cool, that makes sense. I've dropped the base class for now.